e2e/installs.test.js

```diff
@@ -18,21 +20,45 @@ describe('Installation', () => {
     execSync(`rm -rf "${temporaryDirectory}"`);
   });
 
-  beforeEach(() => {
+  beforeEach(async () => {
     appPath = `${temporaryDirectory}/${appName}`;
     execSync(`mkdir ${appPath}`);
+
+    const templateConfig = require('../src/templates/InstantSearch.js/.template.js');
+
+    const config = {
+      template: 'InstantSearch.js',
+      // We fetch the earliest supported version in order to not change
+      // the test output every time we release a new version of a library.
+      libraryVersion: await getEarliestLibraryVersion({
+        libraryName: templateConfig.libraryName,
+        supportedVersion: templateConfig.supportedVersion,
+      }),
+      appId: 'appId',
+      apiKey: 'apiKey',
+      indexName: 'indexName',
+      searchPlaceholder: 'Search placeholder',
+      attributesToDisplay: ['attribute1', 'attribute2'],
+      attributesForFaceting: ['facet1', 'facet2'],
+      organization: 'algolia',
+    };
+
+    configFilePath = `${temporaryDirectory}/config.json`;
+
+    fs.writeFileSync(configFilePath, JSON.stringify(config));
   });
 
   afterEach(() => {
     execSync(`rm -rf "${appPath}"`);
+    execSync(`rm -f "${configFilePath}"`);
   });
 
   describe('Dependencies', () => {
     test('get skipped with the `no-installation` flag', () => {
       execSync(
         `yarn start ${appPath} \
           --name ${appName} \
-          --template "InstantSearch.js" \
+          --config ${configFilePath} \
```

Comment:
This makes the tests easier indeed but we don't test if the CLI options are still asked anymore, which is the main user usage. Can we still cover this behavior somehow?

Reply:
if the cli options are asked, this test would fail. Should there be a non-interactive mode?

Reply:
solved with --no-interactive as a new option, used in the test